lib/prompts/sandboxesPrompt.ts

```diff
@@ -56,17 +62,19 @@ const sandboxTypePrompt = () => {
       default: HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX,
     },
   ]);
-};
+}
 
-const deleteSandboxPrompt = (promptParentAccount = false) => {
+export function deleteSandboxPrompt(
+  promptParentAccount = false
+): Promise<GenericPromptResponse | undefined> {
   const accountsList = getConfigAccounts();
   const choices = promptParentAccount
     ? mapNonSandboxAccountChoices(accountsList)
     : mapSandboxAccountChoices(accountsList);
   if (!choices.length) {
-    return undefined;
+    return Promise.resolve(undefined);
```

Comment:
I don't think you need to do this, can just `return;` and change the return type to `Promise<DeleteSandboxPromptResponse> | void`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/64ec64e3b160db98474cf96fff85513b85b11095